plugin.yaml

```diff
@@ -1,5 +1,5 @@
-molfeat-new:
-  entry_point_prefix: new
-  home_url: ~
-  molfeat_version: ~
+molfeat-MolT5:
```

Comment:
This is a direct PR in molfeat, so no need to fill this since it's not a plugin.

Reply:
I deleted this file